examples/03_fit_predict_plot_midwest_survey.py

```diff
@@ -69,7 +69,7 @@
 encoder_dict = {
     'one-hot': OneHotEncoder(handle_unknown='ignore', sparse=False),
     'similarity': SimilarityEncoder(similarity='ngram'),
-    'minhash': MinHashEncoder(n_components=10, ngram_range=(2, 4),
+    'minhash': MinHashEncoder(n_components=30, ngram_range=(2, 4),
```

Comment:
Same comment on the parameters that take their default values.

Also, in the light of this PR, I suggest to put a default value for n_components, and set it at 30.